code/modules/SCP/SCPs/SCP-2398.dm

```diff
@@ -13,29 +13,29 @@
 	SCP = /datum/scp/scp_2398
 
 /obj/item/weapon/twohanded/scp_2398/attack(mob/living/target,	mob/living/user)
-	var/mob/living/carbon/human/H = target
-	var/mob/living/carbon/human/G = user
+	var/mob/living/carbon/human/T = target
+	var/mob/living/carbon/human/U = user
 	var/activehand = BP_L_ARM //determine which hand to burn (stolen from 113 code because bay code is messy)
-	if(!G.hand)
+	if(!U.hand)
 		activehand = BP_R_ARM
 	if(ishuman(user && target))
```

Comment:
this code never works, replace with individual ishuman checks

Reply:
I think I did it, please guide me.

Reply:
You broke everything

Reply:
What did he break?

Reply:
he forgot an exclamation mark lmao